src/Uno.UWP/Devices/Geolocation/Geolocator.cs

```diff
@@ -24,7 +24,9 @@ public sealed partial class Geolocator
 		private TypedEventHandler<Geolocator, PositionChangedEventArgs> _positionChanged;			
 
 		private PositionAccuracy _desiredAccuracy = PositionAccuracy.Default;
+#if !__ANDROID__
 		private uint? _desiredAccuracyInMeters = DefaultAccuracyInMeters;
+#endif
```

Comment:
Why moving that to other file just for Android?

Reply:
Good question :)
Changing.